main.go

```diff
@@ -40,25 +42,47 @@ func main() {
 	o.Path("/login-complete").Methods("GET").HandlerFunc(handler.LoginComplete)
 
 	a := r.PathPrefix("/api").Subrouter()
-	a.Use(authenticator.Middleware)
+	// a.Use(authenticator.Middleware)
 
 	s := a.PathPrefix("/elections").Subrouter()
-	s.Methods("GET").HandlerFunc(handler.GetElectionIds)
-	s.Methods("POST").HandlerFunc(handler.CreateElection)
+	s.Methods("GET").HandlerFunc(handler.GetElections)
+	s.Path("/create").Methods("POST").HandlerFunc(handler.CreateElection)
 
 	s = a.PathPrefix("/election/{electionId}").Subrouter()
+	s.Methods("GET").HandlerFunc(handler.GetElectionInfo)
+	s.Methods("POST").HandlerFunc(handler.ModifyElection)
```

Comment:
Should this be `POST` or `PUT`? `ModifyElection` sounds like more of a `PUT` operation.

Reply:
PUT makes more sense, I agree